extensions/wikia/AdEngine/js/SlotTweaker.js

```diff
@@ -110,14 +98,29 @@ define('ext.wikia.adEngine.slotTweaker', [
 		}
 	}
 
+	function getRecoveredIframe(slotName) {
+		var fallbackId = win._sp_.getElementId(document.querySelectorAll('div[id="' + slotName + '"] div')[1].id);
```

Comment:
I wonder if it's worth checking if `win._sp_.getElementId(document.querySelectorAll('div[id="' + slotName + '"] div')[1]` exists before accessing it's property